ehr/resources/schemas/dbscripts/postgresql/ehr-21.002-21.003.sql

```diff
@@ -0,0 +1,38 @@
+-- contents of ehr-20.001-20.002.sql
```

Comment:
This comment by itself is a bit confusing... I would update it to explain (briefly) why we're doing this "again"

Reply:
done